contrib/cirrus/lib.sh

```diff
@@ -134,7 +134,7 @@ setup_rootless() {
     # shellcheck disable=SC2154
     if passwd --status $ROOTLESS_USER
     then
-        if [[ $PRIV_NAME = "rootless" ]]; then
+        if [[ $PRIV_NAME = "rootless" ]] && [[ "$TEST_FLAVOR" != "farm"  ]]; then
```

Comment:
Could you explain this exception in a comment? Or better yet remove it?

Reply:
Suggestion:

```
# Farm tests utilize the rootless user to simulate a "remote" podman instance.
# Root still needs to own the repo. clone and all things under `$GOPATH`.  The
# opposite is true for the lower-level podman e2e tests, the rootless user
# runs them, and therefore needs permissions.
```

Reply:
fixed